hudi-common/src/main/java/org/apache/hudi/common/util/collection/BitCaskDiskMap.java

```diff
@@ -452,6 +452,8 @@ private byte[] decompressBytes(final byte[] bytes) throws IOException {
         return decompressBaos.toByteArray();
       } catch (IOException e) {
         throw new HoodieIOException("IOException while decompressing bytes", e);
+      } finally {
```

Comment:
@TengHuo : Thanks for the PR. To be defensive, can you also update line 446 so that the initialization is within the try block. Rest looks good. 
InputStream in = null;
try
{
    in = new InflaterInputStream(new ByteArrayInputStream(bytes));
    ......

Reply:
Sorry for the late replay, quite busy recently.

Got it, thanks for reviewing. Let me add it.

Reply:
Does the code
```java
try (InputStream in = new InflaterInputStream(new ByteArrayInputStream(bytes))) {
}
```

work correctly here?

Reply:
yeah, it is the same thing